var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -33,6 +35,7 @@ class PyPyprecice(PythonPackage):
     version("2.0.0.1", sha256="96eafdf421ec61ad6fcf0ab1d3cf210831a815272984c470b2aea57d4d0c9e0e")
 
     depends_on("precice@develop", when="@develop")
+    depends_on("precice@2.5.0", when="@2.5.0")
     depends_on("precice@2.4.0", when="@2.4.0.1:2.4.0")
```

Comment:
You'll also need to do this for all older versions too:
```suggestion
    depends_on("precice@2.4.0", when="@2.4.0")
```

Reply:
Resolving this for all the past versions made me realize that doing a `for` loop as you previously mentioned is far more easier:

```python
    for ver in ["develop", "2.5.0", "2.4.0", "2.3.0", "2.2.1", "2.2.0", "2.1.1", "2.1.0", "2.0.2", "2.0.1", "2.0.0"]:
        depends_on("precice@" + ver, when="@" + ver)
```
I guess one could create that list in a more fancy way, but for now I would leave it at this.